### PR TITLE
[HOPSWORKS-1804] register monitoring ports with consul

### PR DESCRIPTION
**Pull request by SirOibaf**

### CODE REVIEW

File: templates/default/consul/nm-consul.hcl.erb [hunk 1]
```diff
@@ -42,5 +42,17 @@ services = [
...
+    },
+    {
+        id = "nm/monitoring"
+        name = "nodemanager"
+        tags = [
+            "monitoring"
+        ]
+        port = <%= node['hops']['nm']['metrics_port'] %> 
+        check = {
+            id = "http-health-check"
```
Comment (kouzant) [inline templates/default/consul/nm-consul.hcl.erb:54]: Wrong check id. It should be unique at the machine

File: templates/default/consul/nn-consul.hcl.erb [hunk 2]
```diff
@@ -26,5 +26,17 @@ services = [
...
+    },
+    {
+        id = "nn/monitoring"
+        name = "namenode"
+        tags = [
+            "monitoring"
+        ]
+        port = <%= node['hops']['nn']['metrics_port'] %>
+        check = {
+            id = "nn-http-check"
```
Comment (kouzant) [inline templates/default/consul/nn-consul.hcl.erb:38]: Wrong check id

File: templates/default/consul/rm-consul.hcl.erb [hunk 3]
```diff
@@ -89,5 +89,17 @@ services = [
...
+    },
+    {
+        id = "rm/monitoring"
+        name = "resourcemanager"
+        tags = [
+            "monitoring"
+        ]
+        port = <%= node['hops']['rm']['metrics_port'] %> 
+        check = {
+            id = "rm-http-check"
```
Comment (kouzant) [inline templates/default/consul/rm-consul.hcl.erb:101]: Wrong check id

File: templates/default/consul/nm-consul.hcl.erb (hunk 1 above)
Comment (SirOibaf) [reply templates/default/consul/nm-consul.hcl.erb]: fixed

File: templates/default/consul/nn-consul.hcl.erb (hunk 2 above)
Comment (SirOibaf) [reply templates/default/consul/nn-consul.hcl.erb]: fixed

File: templates/default/consul/rm-consul.hcl.erb (hunk 3 above)
Comment (SirOibaf) [reply templates/default/consul/rm-consul.hcl.erb]: fixed

**Pull request merged**